<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>city_liandan</class>
 <widget class="QWidget" name="city_liandan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>city_liandan</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>285</width>
     <height>350</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="mirror.qrc">:/mirror/Resources/ui/c_0_1.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>24</y>
     <width>15</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="mirror.qrc">
     <normaloff>:/ui/Resources/ui/3.png</normaloff>:/ui/Resources/ui/3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_make">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>306</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
    border-image: url(:/mirror/Resources/ui/c_1_1.png);
}
QPushButton:hover {
	border-image: url(:/mirror/Resources/ui/c_1_1.png);
} 
QPushButton:pressed {
	border-image: url(:/mirror/Resources/ui/c_1_3.png);
} 
QPushButton:disabled {
	border-image: url(:/mirror/Resources/ui/c_1_4.png);
} 
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mm">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>235</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
    border-image: url(:/mirror/Resources/ui/19.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_m1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>74</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
    border-image: url(:/mirror/Resources/ui/20.png);
}
QPushButton:disabled {
	border-image: url(:/mirror/Resources/ui/21.png);
} 
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_m2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>172</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
    border-image: url(:/mirror/Resources/ui/20.png);
}
QPushButton:disabled {
	border-image: url(:/mirror/Resources/ui/21.png);
} 
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_m3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>172</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
    border-image: url(:/mirror/Resources/ui/20.png);
}
QPushButton:disabled {
	border-image: url(:/mirror/Resources/ui/21.png);
} 
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_m5">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>118</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
    border-image: url(:/mirror/Resources/ui/20.png);
}
QPushButton:disabled {
	border-image: url(:/mirror/Resources/ui/21.png);
} 
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_m1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_m2">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>212</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_m3">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>212</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_m4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_m5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_m4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>118</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
    border-image: url(:/mirror/Resources/ui/20.png);
}
QPushButton:disabled {
	border-image: url(:/mirror/Resources/ui/21.png);
} 
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mirror.qrc"/>
 </resources>
 <connections/>
</ui>
